drivers/watchdog/wdt_nrfx.c

```diff
@@ -200,7 +201,25 @@ static const struct wdt_driver_api wdt_nrf_api = {
 
 static int init_wdt(struct device *dev)
 {
+	nrfx_err_t err_code;
+	nrfx_wdt_config_t config;
```

Comment:
`const nrfx_wdt_config_t config = {`
To encourage the compiler to place it in flash.